bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/fileinput/FileInputConfig.java

```diff
@@ -137,6 +141,18 @@ public FileInputConfig maxFileCount(int maxFileCount) {
         return this;
     }
 
+    /** Maximum file size for upload in KB - 0 (default) for no restriction on maximum file size */
+    public FileInputConfig maxFileSize(int maxFileSize) {
+        put(MaxFileSize, maxFileSize);
```

Comment:
Do we need to care about negative values ?

Reply:
> Do we need to care about negative values ?

Isn't an upload of a negative file size actually, mathematically speaking, just a download? Lol :)

Reply:
Would it be better to use wicket-util's `Bytes` instead of `int` as type of the parameter ?
This will also take care of the negative values.

Reply:
I was considering some kind of validation regarding negative values but refrained from it due to the fact that we don't validate other parameters either (e.g. maxFileCount). I didn't know about the `Bytes` implementation - I'll have a look at that option...